### DISCUSSION TITLE
Error deleting organisation if it has no member left.

### DISCUSSION THREAD

**Discussion by Velhenn** [dani-garcia/vaultwarden#4542] [Q&A]
### Error while trying to delete an organisation with no members
An organisation was created for test purposes, to which I was the sole member.
I removed the user from the organisation, then tried to delete it. The admin console returns the following error :
"Error deleting organisation
Error saving organisation"

```
### Environment (Generated via diagnostics page)
* Vaultwarden version: v1.30.5
* Web-vault version: v2024.1.2b
* OS/Arch: linux/x86_64
* Running within a container: true (Base: Debian)
* Environment settings overridden: true
* Uses a reverse proxy: true
* IP Header check: true (X-Real-IP)
* Internet access: true
* Internet access via a proxy: false
* DNS Check: true
* Browser/Server Time Check: true
* Server/NTP Time Check: true
* Domain Configuration Check: true
* HTTPS Check: true
* Database type: SQLite
* Database version: 3.44.0
* Clients used: 
* Reverse proxy and version: 
* Other relevant information: 

```

**Comment by stefan0xC** [comment]
> I removed the user from the organisation

How did you do that? If I try to leave or remove myself from the organization I get an error message that tells me that "The last owner can't leave".

**Reply by stefan0xC** [reply]
The error was probably caused by creating an Organization API Key as described in #4556, so this should be fixed now in `testing`.

**Comment by BlackDex** [comment]
If you manually removed a user, then i suggest to also manually remove the org, because that is the only way this could have happened i think.
